providers/apis/dydx/utils.go

```diff
@@ -58,6 +61,25 @@ var DefaultAPIConfig = config.APIConfig{
 	Endpoints:        []config.Endpoint{{URL: "http://localhost:1317"}},
 }
 
+// DefaultSwitchOverAPIConfig returns the default configuration for the dYdX switch over API provider.
+var DefaultSwitchOverAPIConfig = config.APIConfig{
+	Name:             SwitchOverAPIHandlerName,
+	Atomic:           true,
+	Enabled:          true,
+	Timeout:          20 * time.Second, // Set a high timeout to account for slow API responses in the case where many markets are queried.
+	Interval:         10 * time.Second,
+	ReconnectTimeout: 2000 * time.Millisecond,
+	MaxQueries:       1,
+	Endpoints: []config.Endpoint{
+		{
+			URL: "https://dydx-api.lavenderfive.com", // REST endpoint (HTTP/HTTPS prefix)
+		},
+		{
+			URL: "localhost:9090", // gRPC endpoint (NO HTTP/HTTPS prefix)
```

Comment:
If we're using a 3rd party REST endpoint why are we using a local address for GRPC? I'd suggest that we use local addresses for both

Reply:
had as a placeholder while i was testing. ill just default to 1317 and 9090